assets/src/scss/pages/post/_post.scss

```diff
@@ -28,7 +28,7 @@
   color: $grey-40;
   font-size: 14px;
   font-family: var(--headings--font-family);
-  border-bottom: 1px solid transparentize($grey-20, 0.5);
+  border-bottom: 1px solid rgba(var(--grey-50), 0.5);
```

Comment:
It seems that `--grey-50` is not defined, this doesn't show any border

Reply:
Also, even when replacing `--grey-50` by one that is defined (such as `--grey-500`) the border actually still doesn't show, so maybe these functions (`rgba`, `transparentize`) don't work with CSS variables?

Reply:
Actually why did you switch from `transparentize` to `rgba`? 🤔